app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/view/StatsActivity.kt

```diff
@@ -15,10 +16,24 @@ class StatsActivity : BaseActivity() {
         init()
     }
 
+    private lateinit var binding: ActivityStatsBinding
+
     private fun init() {
         val statsViewModel = ViewModelProviders.of(this).get(StatsViewModel::class.java)
 
-        val binding = DataBindingUtil.setContentView<ActivityStatsBinding>(this, R.layout.activity_stats)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_stats)
         binding.viewModel = statsViewModel
+
+        statsViewModel.mutableHp.observe(this, Observer {
+            if ( it != null) {
+                binding.currentHpLabel.text = Integer.toString(it)
```

Comment:
you can also write `it?.let{...}` instead, I think it looks nicer :)

Reply:
yeah :)